packages/hash/design-system/src/use-scroll-lock.ts

```diff
@@ -10,7 +10,7 @@ import { useLayoutEffect } from "react";
 export const useScrollLock = (active: boolean) =>
   useLayoutEffect(() => {
     document.documentElement.style.cssText = active
-      ? "position: fixed; overflow-y: hidden; width: 100%"
+      ? "position: fixed; overflow-y: scroll; width: 100%"
```

Comment:
What happens if a page is shorter than the viewport? Won’t the scrollbar _appear_ on menu click in this case?